Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -2034,6 +2034,17 @@
     ],
     "percentages-resolve-to": "length"
   },
+  "scrollbar-width": {
+    "affects-layout": false,
+    "animation-type": "by-computed-value",
+    "inherited": false,
+    "initial": "auto",
+    "valid-identifiers": [
```

Comment:
You defined an enum, so this should be `"valid-types": [ "scrollbar-width" ]` instead.

You then might not need some of the other code.

Reply:
I meant instead, not as well. :sweat_smile: